plugins/hydra_ax_sweeper/tests/apps/polynomial_with_coefficients.yaml

```diff
@@ -0,0 +1,22 @@
+# @package _group_
+defaults:
+  - hydra/sweeper: ax
+
+polynomial:
+  # To be minimized
+  coefficients: ???
+
+hydra:
+  sweeper:
+    ax_config:
+      max_trials: 10
+
+      experiment:
+        # Defaults to minimize, set to false to maximize
+        minimize: true
+
+      early_stop:
+        # Number of epochs without a significant improvement from
+        # the currently known best parameters
+        # An Epoch is defined as a batch of trials executed in parallel
+        max_epochs_without_improvement: 2
```

Comment:
since this is for testing, I don't think we need such documentation here.
keep explanatory comments for user examples.